packages/editor/src/lib/options.ts

```diff
@@ -34,7 +34,7 @@ export interface TldrawOptions {
 	readonly collaboratorInactiveTimeoutMs: number
 	readonly collaboratorIdleTimeoutMs: number
 	readonly collaboratorCheckIntervalMs: number
-	readonly cameraMovingTimoutMs: number
```

Comment:
why this change? should we apply this to the others?

Reply:
just a driveby typo fix `Timout` -> `Timeout`

Reply:
ah sorry seems i deleted the `Ms` bit by accident